yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
```

Comment:
Looks like you accidentally `yarn install`d in the root folder probably? Happens all the time, but go ahead and delete this file as we don't need to merge it! Same with the `node_modules` folder in the root folder.

You're gonna wanna run `yarn` within `app/web`. It's a bit confusing as the backend is in python so there's a different command to install packages in the backend.